sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityClientUtilsTests.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.identity;
+
+import com.azure.communication.identity.implementation.models.CommunicationIdentityAccessTokenRequest;
+import com.azure.communication.identity.implementation.models.CommunicationIdentityCreateRequest;
+import com.azure.communication.identity.models.CommunicationTokenScope;
+import com.azure.core.util.logging.ClientLogger;
+import org.junit.jupiter.api.Test;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CommunicationIdentityClientUtilsTests {
+
+    private final ClientLogger logger = new ClientLogger(CommunicationIdentityClient.class);
+
+    @Test
+    public void createCommunicationIdentityCreateRequestWithCustomTokenValidity() {
+        // Arrange
+        int tokenExpirationInMinutes = 60;
+        List<CommunicationTokenScope> scopes = Arrays.asList(CommunicationTokenScope.CHAT);
+        Duration tokenExpiresIn = Duration.ofMinutes(tokenExpirationInMinutes);
+
+        // Action
+        CommunicationIdentityCreateRequest request = CommunicationIdentityClientUtils.createCommunicationIdentityCreateRequest(scopes, tokenExpiresIn, logger);
+
+        // Assert
```

Comment:
assert also whether the right scopes are set, same for other tests(till the 76th line)